### DISCUSSION TITLE
Mojolicious One-liner for serving files

### DISCUSSION THREAD

**Discussion by b03cmans** [mojolicious/mojo#1882] [Ideas]
I am interested on Mojolicious one-liner which servicing static files on specific folder and by default serving the files .

something like : ` python -m http.server 8000`

this one-liner : ` perl -MMojolicious::Lite -MCwd -e 'app->static->paths->[0]=getcwd; app->start' daemon -l http://*:8000`

which could do almost the same but **missing listing the files by default** , is it possible to implement the same feature using Mojolicious one-liner  and enhancing it with option to grep the specific files let's say grepping and serving  only files end with .json

**Comment by Grinnz** [comment]
See https://metacpan.org/pod/App::SimpleHTTPServer which uses https://metacpan.org/pod/Mojolicious::Plugin::Directory::Stylish for directory listings.

**Comment by s1037989** [comment]
This doesn't give you the directory listing you're looking for, but FWIW,
your one-liner can be written more simply:

$ perl -Mojo -e 'app->static->paths->[0]=f; app->start' daemon -l http://
*:8000
